content/intro-curriculum/C_Clean.Rmd

```diff
@@ -298,31 +295,7 @@ head(DO)
 3. Add another line to your code and create a data frame that removes all NA values from `Q_DO`. Woah - that's the same dataframe as our `DO_Q`!
 4. If that goes quickly, feel free to explore other joins (`inner_join`, `full_join`, etc).
 
-##Modify and Summarize
-One area where `dplyr` really shines is in modifying and summarizing. First, we'll look at an example of grouping a data frame and summarizing the data within those groups. We do this with `group_by()` and  `summarize()`. You won't notice much of change between this new data frame and the original because `group_by` is changing the class of the data frame so that `dplyr` handles it appropriately in the next function. Let's look at the average discharge and water temperature by site.
-
-
-```{r group_by_examp, warning=FALSE, message=FALSE}
-class(intro_df)
-
-# Group the data frame
-intro_df_grouped <- group_by(intro_df, site_no)
-class(intro_df_grouped)
-```
-
-Now we can summarize the data frame by the groups established previously.
-
-```{r summarize_examp_NA, warning=FALSE, message=FALSE}
-intro_df_summary <- summarize(intro_df_grouped, mean(Flow), mean(Wtemp))
-intro_df_summary
-```
-
-Notice that this summary just returns NAs. We need the mean calculations to ignore the NA values. We could remove the NAs using `filter()` and then pass that data.frame into `summarize`, or we can tell the mean function to ignore the NAs using the argument `na.rm=TRUE` in the `mean` function. See `?mean` to learn more about this argument.
-
-```{r summarize_examp, warning=FALSE, message=FALSE}
-intro_df_summary <- summarize(intro_df_grouped, mean(Flow, na.rm=TRUE), mean(Wtemp, na.rm=TRUE))
-intro_df_summary
-```
+## Additional functions
 
 There are many other functions in `dplyr` that are useful. Let's run through some examples with `arrange()` and `slice()`.
```

Comment:
I think I would vote for `arrange` getting moved to explore as well.  What do you think?